libs/obs/include/mrpt/obs/CObservationVelodyneScan.h

```diff
@@ -200,7 +200,7 @@ class CObservationVelodyneScan : public CObservation
 		/** The list of point indices (in x,y,z,...) generated for each laserID
 		 * (ring number). Generated only if `generatePointsForLaserID`=true in
 		 * `TGeneratePointCloudParameters`*/
-		std::vector<std::vector<std::size_t>> pointsForLaserID;
+		std::vector<std::vector<uint64_t>> pointsForLaserID;
```

Comment:
@jlblancoc This one fixes a 32-bit/64-bit compatibility issue, but files saved on 32-bit machines will be broken.